helm/provisioner/values.yaml

```diff
@@ -60,6 +60,10 @@ common:
   #
   #labelsForPV:
   #  pv-labels: can-be-selected
+  #
+  # Indicates if Windows DaemonSet should be created (by default: true)
+  #
+  enableWindows: true
```

Comment:
Maybe the default value should be false to make Windows deployment an opt-in, unfortunately the chart was already released so I'd argue this'd be a breaking change.

@msau42 @andyzhangx thoughts? I don't know how many people are relying on the helm chart to install the Windows component, as far as I know it was only Microsoft that's using it, maybe we can make this change in a minor patch (e.g. release 1.1.0 of the helm chart).

Reply:
It looks like perhaps making `default value = true` makes most sense here, for obvious reasons (not introducing breaking changes etc) 

In this case, perhaps `disableWindows` naming would make more sense here, since by default we are providing `linux` and `windows` DaemonSet.  

What do you guys think? @mauriciopoppe @andyzhangx @msau42 
If you agree, I would change this to `disableWindows` to allow users to disable Windows if for some users who won't need Windows DaemonSets. Also, adding documentation to `sig-storage-local-static-provisioner/helm/README.md` for this new feature, to allow users to disable windows deployment.

Reply:
@msau42 @andyzhangx @mauriciopoppe What do you think guys? Is this feature necessary, considering there're users asking for disabling windows deployment for its redundancy.

Reply:
yeah I think we can add it however we need to be ok whether it should be enabled by default

Reply:
@andyzhangx @msau42 please give your valuable opinions here for this feature addition. 

Anything would happen if we don't enableWindows by default? How many users would be affected? 

Or maybe perhaps we add the feature to disableWindows. But it is counterintuitive in nature....considering all features in helm values are addition add-ons, rather than removals...

Dilemma here.

Reply:
I think we can make it `enableWindows: false` by default but we'd need to bump the Helm Chart to v2 because there might be people who already rely on this manifest being installed by default. The more we wait the harder it's going to make this switch.

@andyzhangx are you installing LVP through the Helm chart?

Reply:
@mauriciopoppe Alright lets change to `enableWindows: false` by default. I have changed the necessary and also added into documentation as well.